app/services/most_recent_content.rb

```diff
@@ -0,0 +1,37 @@
+class MostRecentContent
+  attr_reader :content_id, :filter_content_purpose_supergroup, :number_of_links
+
+  def initialize(content_id:, filter_content_purpose_supergroup:, number_of_links: 5)
+    @content_id = content_id
+    @filter_content_purpose_supergroup = filter_content_purpose_supergroup
+    @number_of_links = number_of_links
+  end
+
+  def self.fetch(content_id:, filter_content_purpose_supergroup:)
+    new(content_id: content_id, filter_content_purpose_supergroup: filter_content_purpose_supergroup).fetch
+  end
+
+  def fetch
+    search_response.documents
+  end
+
+private
+
+  def search_response
+    search_fields = %w(title
```

Comment:
This is identical to the array of fields above in MostPopularContent (line 22) consider making a constant available to both.